### PR TITLE
Added the document of ByConity TSC

### LINKED ISSUE #98: ByConity Technical Steering Committee (TSC)
## Definition of the ByConity developement roles 
* **Users:** A user is someone who uses our software. They contribute to Apache projects by providing feedback to developers in the form of bug reports and feature suggestions. 
* **Contributors:** Include anyone in the community that contributes code, documentation, pull request reviews, designs, or other technical artifacts to the Project (“Contributors”);
* **Committers:** Are Contributors who have earned the ability to modify (“commit”) source code, documentation, or other technical artifacts in a project’s repository (“Committers”); 
* **Maintainers:** a committer who was elected due to merit for the evolution of the project. They have write access to the code repository, the right to vote on community-related decisions and the right to propose other active contributors for committership. 

## ByConity Technical Steering Committee
The ByConity Technical Steering Committee (TSC) is a group of **maintainers** that contribute to the maintenance of the ByConity Initiative. It is a group of people that helps make decisions base on the best interesting the community. The TSC members commit to participate in voting process.
 
## We can take the below documents as a reference 

* [Asyncapi TSC membership](https://github.com/asyncapi/community/blob/master/TSC_MEMBERSHIP.md)
* [How the ASF works](https://www.apache.org/foundation/how-it-works.html)

### PR DESCRIPTION
**Pull request by WillemJiang**
Fix #98

### CODE REVIEW

File: docs/community/tsc-membership.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* Asynchronous decision making 
+* Every maintainer is allowed to automatically become a TSC member
+
+## Benefits of becoming a TSC member
+
+As a TSC member you will enjoy the following benefits: 
+ 
+* Recognition from ByConity community
+* Participation in the voting process on topics concerning the entire initiative
+* Included in key decision-making for the Initiative.
```
Comment (Adora627) [inline docs/community/tsc-membership.mdx:43]: Many developers are caring about Roadmap and its related decision. Shall we clarify how are we going to do with it?

File: docs/community/tsc-membership.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+As a TSC member you will enjoy the following benefits: 
+ 
+* Recognition from ByConity community
+* Participation in the voting process on topics concerning the entire initiative
+* Included in key decision-making for the Initiative. 
+
+## How to become a TSC member
+
+Any maintainer can become a TSC member. To become a TSC member, you need to become a maintainer of one of the projects in the ByConity Initiative.
+
```
Comment (Adora627) [inline docs/community/tsc-membership.mdx:48]: I'm a little confused about what roles can be called as TSC member? Only maintainer or other roles(i.e. contributor, commitor) also are TSC member?

**Review by Adora627 — CHANGES_REQUESTED** [review]

File: docs/community/tsc-membership.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+benefits of being a TSC member, and the steps to become a TSC member.
+
+## Development Roles in ByConity
+* **Users:** Individuals who use ByConity's software and provide feedback in the form of bug reports and feature suggestions. 
+* **Contributors:** Members of the community who contribute code, documentation, pull request reviews, designs, 
+or other technical artifacts to the ByConity initiative.
+* **Committers:** Contributors who have earned the ability to modify ("commit") source code, documentation, 
+or other technical artifacts in a project's repository.
+* **Maintainers:** Committers who have been elected due to their merit and contributions towards the development and evolution of the project. 
+They have write access to the code repository, can vote on community-related decisions, and can propose other active contributors for committership.
```
[the remark this answers is not in the document]
Comment (WillemJiang) [reply docs/community/tsc-membership.mdx:20]: It relates to how much effort we can put into the project. A maintainer must know the code history well and make good design decisions. I double we can have more than 9 maintainers so far. 
For the TSC members, we can set up the cap to 5 and extend it to 9 if more developers are growing.

File: docs/community/tsc-membership.mdx (hunk 1 above)
Comment (WillemJiang) [reply docs/community/tsc-membership.mdx:43]: We can start the roadmap proposal discussion in the GitHub issues to collect the feedback from the community to hear more voices from the developers.  Then we can make a decision base on this feedback.

File: docs/community/tsc-membership.mdx (hunk 2 above)
Comment (WillemJiang) [reply docs/community/tsc-membership.mdx:48]: No, a TSC member must be a maintainer, but a maintainer may not be a TCS member.

**Review by Adora627 — APPROVED** [review]

**Review by Adora627 — APPROVED** [review]

**Pull request merged**